lms/static/sass/shared/_footer.scss

```diff
@@ -301,13 +306,23 @@ $edx-footer-bg-color: rgb(252,252,252);
     @extend %edx-footer-reset;
     @extend %edx-footer-section;
     width: flex-grid(6,12);
+
+    @media print {
+      width: 100%;
+      border: none;
+      padding-right: none;
+      margin-right: none;
```

Comment:
Ah yeah, I commented on that inline as well, but github collapsed the change: https://github.com/edx/edx-platform/pull/7595#discussion_r30268028

We just want to make sure it is margin-right(0); instead of "none" but +1